third_party/ipam/README

```diff
@@ -0,0 +1,9 @@
+Packages below have been copied from [k8s.io/kubernetes@/v1.21.1](https://github.com/kubernetes/kubernetes/tree/v1.21.1) to avoid importing the whole kubernetes repo:
```

Comment:
I assume you didn't make any change to these files and copied them "as is". Otherwise it would be good to mention the major changes here.